Homework-6/Program.cs

```diff
@@ -0,0 +1,40 @@
+using Homework_6.Service;
+
+namespace Homework_6
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            var builder = WebApplication.CreateBuilder(args);
+
+            // Add services to the container.
+
+            builder.Services.AddControllers();
+            // Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+
+            builder.Services.AddSingleton<StudentService>();
```

Comment:
StudentService should be Scoped where a new instance of the service is created per scope.
With Singleton, a single instance of the service is created and shared throughout the application's lifetime